fesvr/htif.h

```diff
@@ -11,6 +11,16 @@
 #include <map>
 #include <vector>
 #include <assert.h>
+#include <iostream>
+
+static volatile bool signal_exit = false;
```

Comment:
It doesn't seem like a good idea to define global variables like this in a header file. Everyone who includes it will get their own copy.